internal/kms/kes.go

```diff
@@ -138,9 +139,30 @@ func NewWithConfig(config Config) (KMS, error) {
 			}
 		}
 	}()
+
+	go func() {
+		c.keepKeyInCache()
+	}()
 	return c, nil
 }
 
+// Request KES keep an up-to-date copy of the KMS master key to allow minio to start up even if KMS is down. The
+// cached key may still be evicted if the period of this function is longer than that of KES .cache.expiry.unused
+func (c *kesClient) keepKeyInCache() {
+	ctx := context.Background()
+
+	ticker := time.NewTicker(10 * time.Second)
```

Comment:
```suggestion
	ticker := time.NewTicker(10 * time.Second)
```

Please make this configurable via MINIO_KMS_KEY_CACHE_INTERVAL=10s

Reply:
Do not use a ticker because ValidateKey()'s response time would account for the ticker firing, causing misfires.

Use a timer instead to check how `cmd/erasures-sets.go` 

```
// cleanup ".trash/" folder every 5m minutes with sufficient sleep cycles, between each
// deletes a dynamic sleeper is used with a factor of 10 ratio with max delay between
// deletes to be 2 seconds.
func (s *erasureSets) cleanupDeletedObjects(ctx context.Context) {
        timer := time.NewTimer(globalAPIConfig.getDeleteCleanupInterval())
        defer timer.Stop()

```

Reply:
Implemented